ctapipe/resources/train_disp_reconstructor.yaml

```diff
@@ -0,0 +1,71 @@
+# ======================================================================
+# ctapipe-train-disp-reconstructor config file.
+# version: VERSION
+#
+# Configuration for training machine-learning models
+# ======================================================================
+TrainDispReconstructor:
+  CrossValidator:
+    n_cross_validations: 5
+
+  DispReconstructor:
+    norm_cls: ExtraTreesRegressor
+    norm_config:
+      n_estimators: 10
+      max_depth: 10
+      n_jobs: -1
+
+    log_target: True
+
+    sign_cls: ExtraTreesClassifier
+    sign_config:
+      n_estimators: 10
+      max_depth: 10
+      n_jobs: -1
+
+    features:
+      - hillas_intensity
+      - hillas_r
+      - hillas_length
+      - hillas_width
+      - hillas_skewness
+      - hillas_kurtosis
+      - timing_slope
+      - timing_deviation
+      - leakage_intensity_width_1
+      - leakage_intensity_width_2
+      - leakage_pixels_width_1
+      - leakage_pixels_width_2
+      - concentration_cog
+      - concentration_core
+      - morphology_n_islands
+      - morphology_n_large_islands
+      - morphology_n_medium_islands
+      - morphology_n_pixels
+      - morphology_n_small_islands
+      - HillasReconstructor_tel_impact_distance
```

Comment:
Should I remove the predicted energy as well, since the stereo parameters are used for that meaning mono events would not be predicable if the energy prediction is used here?